### DISCUSSION TITLE
Can’t update sources

### DISCUSSION THREAD

**Discussion by seperior82** [wstyres/Zebra#1977] [Q&A]
Hello! When I try to update sources it keeps running and running but never updates them. I have cleared sources cache but the same issue remains, any help?

**Comment by iAdam1n** [comment]
How many sources do you have installed?

**Reply by seperior82** [reply]
151, but never had any problem. Last day I updated sources was the 24th of april but when I tried today this issue was present.

**Reply by iAdam1n** [reply]
There is an issue we're aware of if you have many sources but if it worked before then maybe just some if the sources are taking longer than usual.

**Reply by seperior82** [reply]
I don't think it's taking longer than usual because I had them updating for 15 minutes and they don't update.

**Reply by D1mar0n** [reply]
Hey! I have the same problem since yesterday. Nothing is updated.

**Reply by seperior82** [reply]
So it's not just me 🤔

**Reply by D1mar0n** [reply]
> So it's not just me 🤔

You are not alone with this problem...

**Reply by D1mar0n** [reply]
Everything just started working again!

**Reply by seperior82** [reply]
Really? I'll go and check

**Reply by seperior82** [reply]
Thank god 🙏🏼 your right, they updated super fast as usual again!! Yaaayyy 🎉🎉🎉

**Comment by D1mar0n** [comment]
Отправлено с iPhone
